posixlib/src/main/scala/scala/scalanative/posix/netdb.scala

```diff
@@ -50,6 +50,8 @@ object netdb {
 
   @name("scalanative_AI_CANONNAME")
   def AI_CANONNAME: CInt = extern
+
+  def gai_strerror(code: CInt): CString = extern
```

Comment:
If the gai_strerror() declaration moved up to just before
```
 @name("scalanative_getaddrinfo")
  def getaddrinfo(name: CString,
```
then the methods & "constants" would then each be grouped together
rather than interspersed.  The methods would also be in alphabetical order
and easier to find.

Reply:
Agreed that is a good idea.